audformat/core/attachment.py

```diff
@@ -59,6 +60,38 @@ def __init__(
         self.path = path
         r"""Attachment path"""
 
+    @property
+    def files(
+            self,
+    ) -> typing.List[str]:
+        r"""List all files part of the attachment.
+
+        List recursively the relative path
+        of all files that exist
+        under :attr:`audformat.Attachment.path`
+        on hard disc.
+
+        """
+        files = []
+        if not self._db.root:
+            return files
+
+        path = audeer.path(self._db.root, self.path)
+        if not os.path.exists(path):
+            return files
+
+        if os.path.isdir(path):
+            files = audeer.list_file_names(path, recursive=True)
```

Comment:
Will this also include symbolic links?

Reply:
It will include symbolic links in the returned list of files. I added a check for this and raise now an error if any file is a symbolic link.